packages/docs/gatsby-config.js

```diff
@@ -1,3 +1,10 @@
+const path = require("path");
+
+console.log(
```

Comment:
this ########### necessary?

Reply:
Fair but a minor point that we probably don't want to delay the release for